test/integration/full/aria-hidden-body/pass.html

```diff
@@ -26,6 +26,6 @@ <h2>Some title.</h2>
     <div id="mocha"></div>
     <script src="pass.js"></script>
     <script src="/test/integration/adapter.js"></script>
-    <iframe src="fail.html"></iframe>
```

Comment:
This creates a race condition, between fail.js calling axe.run, and pass.js calling collectResultFromFrame.